porter/v1/eks.proto

```diff
@@ -72,3 +76,9 @@ message EKSLogging {
   bool enable_audit_logs = 4;
   bool enable_controller_manager_logs = 5;
 }
+
+//EKSPreflightValues contains all needed values to perform EKS Preflight Checks
+message EKSPreflightValues {
+  ////EKS contains all information required to configure the EKS cluster's preflight checks
+  EKS eks = 1;
```

Comment:
I would split this out separately, and not change the entire cluster contract for building EKS clusters, just for preflight checks. Duplicate fields if you have to.

Reply:
Done thanks